File: MusicPlayer.DAL/Entities/Playlist.cs
using System.Collections.Generic;

namespace MusicPlayer.DAL.Entities
{
    public class Playlist : BaseEntity<int>
    {
        public string Name { get; set; }

        public string Summary { get; set; }

        /// <summary>
        /// Обложка
        /// </summary>
        public string CoverUrl { get; set; }

        public List<Track> Tracks { get; set; }
    }
}

File: clientapp/src/models/PlayList.ts
import Track from "./Track";

export default class Playlist {
  id: number;
  name: string;
  summary: string;
  coverUrl: string;
  tracks: Track[];

  constructor() {
    this.id = 0;
    this.name = "";
    this.summary = "";
    this.coverUrl = "";
    this.tracks = [];
  }
}
